packages/integration-tests/legacy-suite/test-case/case.js

```diff
@@ -1,15 +1,46 @@
 (async () => {
```

Comment:
I think we should separate E2E tests into folders

Reply:
Thanks for this enhancement. What do you think especially for this PR?

Reply:
It's not directly related to the topic of this PR, I just noticed that original content of `legacy-suite` has been replaced with a mixture of commented and uncommented tests.

Since we already talked about separate E2E suites, I see these changes as a regression

Reply:
Well i saw it and resolved it in https://github.com/sakuli/sakuli/pull/137/commits/104f6eb258f45f8c195fdf764aa19d11740f8579